### PR TITLE
Make webmozarts/console-parallelization optional

### LINKED ISSUE #11749: Make webmozarts/console-parallelization optional

### PR DESCRIPTION
**Pull request by dvesh3**
## Changes in this pull request  
Resolves #11749

### CODE REVIEW

File: phpstan-parameters.neon [hunk 1]
```diff
@@ -18,6 +18,7 @@ parameters:
         - '**/PDFreactor.class.php'
         - '**/bundles/AdminBundle/src/Security/Guard/AdminAuthenticator.php'
         - '**/bundles/AdminBundle/src/Security/Authenticator/AdminAbstractAuthenticator.php'
+        - '**/ParallelizationBase.php'
```
Comment (blankse) [inline phpstan-parameters.neon]: Suggest Packages should be available in Tests. Should be also added to require-dev

Comment (dvesh3) [reply phpstan-parameters.neon]: Done.

File: lib/Console/Traits/ParallelizationBase.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            \Webmozarts\Console\Parallelization\Parallelization::execute as parentExecute;
+        }
+    }
+} else {
+    trait ParallelizationBase
+    {
+        /**
+         * {@inheritdoc}
+         */
+        public function execute(InputInterface $input, OutputInterface $output): void
```
Comment (aryaantony92) [inline lib/Console/Traits/ParallelizationBase.php]:
```suggestion
        public function execute(InputInterface $input, OutputInterface $output): int

```

File: lib/Console/Traits/ParallelizationBase.php [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            foreach ($items as $item) {
+                $this->runSingleCommand(trim($item), $input, $output);
+            }
+
+            //Method executed after executing all the items
+            if (method_exists($this, 'runAfterBatch')) {
+                $this->runAfterBatch($input, $output, $items);
+            }
+
+            $this->runAfterLastCommand($input, $output);
```
Comment (aryaantony92) [inline lib/Console/Traits/ParallelizationBase.php:56]: return 0;

Comment (aryaantony92) [inline lib/Console/Traits/ParallelizationBase.php:57]:
```suggestion
            $this->runAfterLastCommand($input, $output);
            return 0;
```

**Pull request merged**